app/cdap/components/FooterPanel/Components/TableMetaInfoTab/index.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import RenderLabel from 'components/FooterPanel/Components/common/RenderLabel';
+import TabWrapper from 'components/FooterPanel/Components/common/TabWrapper';
+import { ITableMetaInfoTabProps } from 'components/FooterPanel/Components/TableMetaInfoTab/types';
```

Comment:
This component is very small and the type is very simple - no need to split out the type into a separate file unless its used somewhere else or there are a lot of types and this file is too big already

Reply:
Done